src/main/java/no/sikt/sws/IndexHandler.java

```diff
@@ -33,11 +34,19 @@ Pair<String, Integer> inputToOutputWithStatus(String body, RequestInfo request,
         try {
             var url = workspace + "-" + resourceIdentifier;
             logger.info("URL: " + url);
-            var response = openSearchClient.sendRequest(httpMethod, url, body);
+
+            var response = openSearchClient.sendRequest(
+                    httpMethod,
+                    url,
+                    WorkspaceStripper.add(body, workspace, resourceIdentifier));
```

Comment:
Can you revert this? As its out of scope of this change where we focus on stripping response body?